CONTRIBUTING.md

```diff
@@ -117,7 +117,9 @@ hard-linked extension modules, and the builtins. The `VERSIONS` file lists
 the versions of Python where the module is available.
 
 We accept changes for future versions of Python after the first beta for that
-version was released.
+version was released. We will drop active support for a Python version three
```

Comment:
Maybe something like "cease actively testing support for a Python version"? The stubs for that version will probably continue to be entirely accurate in the three months when the version is untested but the branches are still there

Reply:
We do more than that, though. We will declare end of support in the docs. I don't think we need to be completely accurate, especially since this policy is mostly relevant to typeshed and type checker developers. End users are only indirectly affected through their type checkers.

Reply:
Fair enough. I'd prefer to be precise here since these CONTRIBUTING docs are already not exactly concise, so I don't think the marginal impact of another sentence is particularly big! But I don't have a strong opinion.

Reply:
Agree that it's useful to be precise about our versioning policy, so we don't have to wrangle over it again next year when we start to think about dropping 3.9.

Reply:
My comment was contradictory anyway. Precision is good, but I want to word it in a way that makes it clear that official support is dropped after 2-3 months.